services/wfm/src/main/java/org/openkilda/wfm/topology/flowhs/bolts/SpeakerWorkerBolt.java

```diff
@@ -74,6 +74,14 @@ public void onTimeout(String key, Tuple tuple) throws PipelineException {
         service.handleTimeout(key);
     }
 
+    @Override
+    protected void unhandledInput(Tuple input) {
+        if (log.isTraceEnabled()) {
```

Comment:
why is it trace?

Reply:
So as a designed for now both create and reroute "hubs" have their own workers, which are receiving tuples from one spout. Currently, we haven't found a solution for it, so receiving "unexpected" tuples is expected and there is no need to flood logs, but we keep trying to find better solution